ha/k8s_setup/ha_start.py

```diff
@@ -53,15 +67,19 @@ def handle_signal(signum, frame):
     signal.signal(signal.SIGSEGV, handle_signal)
     signal.signal(signal.SIGTERM, handle_signal)
 
-    driver_process = Popen(['/usr/bin/python3', service_entry_mapping[args.services]], shell=False, stdout=PIPE, stderr=STDOUT) # nosec
-
-    sys.stdout.write(f"The driver process with pid {driver_process.pid}, and args {driver_process.args} started successfully.")
+    start_driver_process()
 
     # Poll process.stdout to show stdout live
     while True:
         output = driver_process.stdout.readline()
         if driver_process.poll() is not None:
-            break
+            if((args.debug&AUTO_RESTART)!=0):
+                if output:
```

Comment:
Can u please add space before and after != ., and no required outer bracket.

Reply:
these chnages are not to be part of this PR so removed